Zend/zend_opcode.c

```diff
@@ -629,6 +630,19 @@ ZEND_API int pass_two(zend_op_array *op_array)
 		CG(context).literals_size = op_array->last_literal;
 	}
 	opline = op_array->opcodes;
+
+	if ((op_array->fn_flags & ZEND_ACC_PUBLIC) != 0
+		&& (op_array->fn_flags & ZEND_ACC_ABSTRACT) == 0
+		&& op_array->required_num_args == 0
+		&& op_array->last == 3
+		&& opline->opcode == ZEND_FETCH_OBJ_R 
+		&& opline->op1_type == IS_UNUSED 
+		&& opline->op2_type == IS_CONST
+		&& (opline+1)->opcode == ZEND_RETURN)
```

Comment:
Is there a quick way to detect EXT op and Non-EXT? or I just check each separated op?

And I guess the actual question is: should we optimize the code when zend extensions like xdebug is on?